bananas/admin/api/schemas/decorators.py

```diff
@@ -0,0 +1,20 @@
+from functools import wraps
+
+
+def navigation(view):
```

Comment:
Think we should remove this navigation-specific decorator in favour of the more generic below.